src/test/scala/com/databricks/spark/redshift/QueryTest.scala

```diff
@@ -19,20 +19,18 @@ package com.databricks.spark.redshift
 
 import org.apache.spark.sql.catalyst.plans.logical
 import org.apache.spark.sql.{Row, DataFrame}
+import org.scalatest.FunSuite
 
 /**
- * Copy of Spark SQL's `QueryTest` object.
+ * Copy of Spark SQL's `QueryTest` trait.
  */
-object QueryTest {
+trait QueryTest extends FunSuite {
```

Comment:
+1 to this change.  I much prefer this style to having companion objects, since invocation is less verbose and its less error prone to let checkAnswer actually assert the check.  I'd like to change Spark back as well.